src/index.html

```diff
@@ -1,13 +1,433 @@
+  <!-- #region head -->
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>THE MET</title>
+    <link
+    rel="icon"
+    type="image/x-icon"
+    href="./images/logo-2x.png"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Cinzel:wght@400;700&family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/style.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page_body">
+    <header class="header">
+      <div class="container">
+        <div>
+          <div class="top-bar">
+            <a class="top-bar__logo-link" href="#">
+              <img class="top-bar__logo" src="./images/logo-2x.png"
+              alt="The Met Logo">
```

Comment:
Need to check and fix it everywhere
```suggestion
              <img
                class="top-bar__logo"
                src="./images/logo-2x.png"
                alt="The Met Logo"
              >
```